src/components/Cleanups/CleanupList.vue

```diff
@@ -0,0 +1,55 @@
+
+  <template>
+<div class="container">
+  <h3>Active Oakland Cleanups</h3>
+  <div class="list-group"
+      v-for="cleanup in cleanupList"
+      v-bind:cleanups="cleanup"
+      v-bind:key="cleanup.id">
+      <a class='list-group-item list-group-item-action' href='#'>
+        <div class='row'>
+          <img class='img-fluid img-thumbnail col-4' src="http://127.0.0.1:8000/assets/images/crossed_rakes.png"/>
```

Comment:
Store the images locally and use the path, This will not work anywhere other than 127.0.0.1

Reply:
Do we have the api endpoint stored somewhere on the Vue object? Here are the base URLs:

- API:  `https://trashtalkdev.herokuapps.com/api/v1/` 
- Static files: `https://trashtalkdev.herokuapps.com/static/images/`